onedal/_device_offload.py

```diff
@@ -194,17 +189,30 @@ def support_usm_ndarray(freefunc=False, queue_param=True):
 
     def decorator(func):
         def wrapper_impl(obj, *args, **kwargs):
-            usm_iface = _extract_usm_iface(*args, **kwargs)
+            data = (*args, *kwargs.values())
+            if len(data) == 0:
+                return _run_on_device(func, obj, *args, **kwargs)
```

Comment:
```suggestion
            if len(args) == 0 and len(kwargs) == 0:
                return _run_on_device(func, obj)
```
Rather than generate a tuple for no reason just for an if check, and if there are no elements, drop them.

Reply:
Yeah, good point. Could be shorter